schema/defs.json

```diff
@@ -41,10 +41,10 @@
             "minimum": 0,
             "maximum": 18446744073709552000
         },
-        "percent": {
+        "maximum": {
             "type": "integer",
             "minimum": 0,
```

Comment:
Looking at this again. I think this is actually a generic percent definition, rather than specific to CPU Percent (Oops), so I should not touch it. Am I reading that right?